submissions/smillims/oop/app.js

```diff
@@ -0,0 +1,120 @@
+const block = {
+	width: 101,
+	height: 85
+};
+const positionEnemyY = [60, 140, 220];
+const positionPlayer = {
+	playerX: 200,
+	playerY: 400,
+};
+
+const widthEnemies = [80];
```

Comment:
Is there any good reason to have an array here? What will happen if we add yet another value in the list?

Reply:
Oh, then would be some troubles. I'm:
1) Decrease value
2) Change variable